test/object/SYLT.mjs

```diff
@@ -0,0 +1,84 @@
+import { deepStrictEqual } from 'assert';
+import { getEmptyBuffer, id3Header } from '../utils.mjs';
+import { encodeWindows1252, encodeUtf16le } from '../../src/encoder.mjs';
+import ID3Writer from '../../dist/browser-id3-writer.js';
+
+describe('SYLT', () => {
+  it('SYLT', () => {
+    const writer = new ID3Writer(getEmptyBuffer());
+    writer.padding = 0;
+    writer.setFrame('SYLT', {
+      text: [
+        ["She's up all night 'til the sun", 1],
+        ["I'm up all night to get some", 2],
+        ["She's up all night for good fun", 3],
+        ["I'm up all night to get lucky", 4],
+      ],
+      type: 1,
+      timestampFormat: 2,
+      language: 'eng',
+    });
+    writer.addTag();
+    const actual = new Uint8Array(writer.arrayBuffer);
+    const expected = new Uint8Array([
+      ...id3Header,
+      0,
+      0,
+      2,
+      34, // tag size without header
+      ...encodeWindows1252('SYLT'),
+      0,
+      0,
+      1,
+      24, // size without header
+      0,
+      0, // flags
+      1, // text encoding
+      ...encodeWindows1252('eng'), // language
+      2, // Time stamp format
+      1, // Content type
+      0xff,
+      0xfe, // BOM
+      ...encodeUtf16le(''), // description (empty)
```

Comment:
could you also add here not empty description to check if it is properly encoded?